testsuite/MDAnalysisTests/guesser/test_default_guesser.py

```diff
@@ -0,0 +1,236 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4 fileencoding=utf-8
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+# doi: 10.25080/majora-629e541a-00e
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+import pytest
+from pytest import approx
+import MDAnalysis as mda
+
+from MDAnalysis import guesser
+from numpy.testing import assert_equal, assert_allclose
+import numpy as np
+from MDAnalysis.core.topologyattrs import (Angles, Atomtypes, Atomnames)
+from MDAnalysis.guesser.default_guesser import DefaultGuesser
+from MDAnalysis.core.topology import Topology
+from MDAnalysisTests import make_Universe
+from MDAnalysisTests.core.test_fragments import make_starshape
+import MDAnalysis.tests.datafiles as datafiles
+
+from MDAnalysisTests.util import import_not_available
+
+try:
+    from rdkit import Chem
+    from rdkit.Chem.rdPartialCharges import ComputeGasteigerCharges
+except ImportError:
+    pass
+
+requires_rdkit = pytest.mark.skipif(import_not_available("rdkit"),
+                                    reason="requires RDKit")
+
+
+@pytest.fixture
+def default_guesser():
+    return DefaultGuesser(None)
+
+class TestGuessMasses(object):
+    def test_guess_masses_from_universe(self):
+        topology = Topology(3, attrs=[Atomtypes(['C', 'C', 'H'])])
+        u = mda.Universe(topology)
+        u.guess_TopologyAttributes(to_guess=['masses'])
```

Comment:
Add a comment, then. Otherwise, your intent will be lost very quickly.

In general, it is very common, especially in tests, to think about things you want to account for that are not obvious when reading the code. These are the most important places to write comments: when you are not describing the code but your intentions.

Reply:
indeed.... doesn't make sense to me now tbh, so I removed it